src/misc/sw-template.js

```diff
@@ -278,6 +279,38 @@ function WebpackServiceWorker(params) {
     // * urlString -- url to match from the CACHE_NAME
     // * event.request -- original Request to perform fetch() if necessary
 
+    if (responseStrategy === "network-first") {
+      event.respondWith(
+        fetch(event.request)
+        .then((response) => {
+          if (DEBUG) {
+            console.log('[SW]:', `URL [${ urlString }] from network`);
+          }
+
+          // always fresh cache response for future usage
```

Comment:
I'm not really sure that we need to refresh existing for requests gotten from the network. The problem is that if we use `updateStrategy=change` we would need to wipe hash associations of the assets being request, because we don't know if network returned new version or not.

The easies way to handle all this and avoid collisions (with current approach one file can be v3 and another v10)  is by simply updating assets in a caches only when `install` even happens, as it's now. 

So the code would be something like this:

```js
if (response.ok) {
  return response;
} else {
  return cachesMatch(request, CACHE_NAME)
}
```